.changeset/config.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://unpkg.com/@changesets/config@1.7.0/schema.json",
   "changelog": "@changesets/cli/changelog",
   "commit": false,
-  "linked": [["@xata.io/client", "@xata.io/codegen"]],
+  "linked": [["@xata.io/client", "@xata.io/codegen", "@xata.io/importer"]],
```

Comment:
Not sure if we want to link all dependency versions. The codegen kind of made sense, but maybe no need to bump all minor/major changes for all of them at the same time.